devel/nodejs18/Portfile

```diff
@@ -43,12 +43,10 @@ depends_build-append    port:pkgconfig
 set py_ver              3.10
 set py_ver_nodot        [string map {. {}} ${py_ver}]
 
-depends_lib-append      port:python${py_ver_nodot} \
+depends_lib-append      port:icu \
```

Comment:
Just one complaint, the dependency should be: `path:lib/pkgconfig/icu-uc.pc:icu`

Reply:
Why should that be the dependency? As I read it, this wou ldallow icu58 also? This version of nodejs requires ICU 68 or newer, so if we allow anything providing `lib/pkgconfig/icu-uc.pc` nodejs18 won't compile against it and the build will fail.

Reply:
> Why should that be the dependency? As I read it, this wou ldallow icu58 also? This version of nodejs requires ICU 68 or newer, so if we allow anything providing `lib/pkgconfig/icu-uc.pc` nodejs18 won't compile against it and the build will fail.

No, `icu58` is wholly segregated from `icu`, with the contents installed under `${prefix}/libexec/icu58`. Including the pkgconfig file:

```
/opt/local/libexec/icu58/lib/pkgconfig/icu-uc.pc
```

Reply:
> Why should that be the dependency?

And we're using a path dep, to support a Devel port for `icu`. It doesn't exist yet, but it likely will sooner rather than later.

That's why everything else uses the dep I provided.

Reply:
Ok thanks for explaining. I just pushed a new version using `path:lib/pkgconfig/icu-uc.pc:icu`.